packages/client/src/entities/user/model/converters.ts

```diff
@@ -0,0 +1,16 @@
+import { AvatarForm } from "@/features/Profile/change-avatar/types";
+import { PasswordForm } from "@/features/Profile/change-password/types";
+
+import { UserPassword } from "./types";
+
+export const passwordConverter = (data: PasswordForm): UserPassword => ({
+  oldPassword: data.oldPassword,
+  newPassword: data.password,
+});
+
+export const avatarConverter = (data: AvatarForm): FormData => {
+  const formData = new FormData();
+  formData.append("avatar", data.avatar!);
```

Comment:
Может быть не стоит использовать оператор '!'? Это потом порождает ошибки, которые не отслеживаются тайпскриптом. Здесь можно сделать так
```suggestion
if (data.avatar) {
  formData.append("avatar", data.avatar);
}
```

Reply:
Данный конвертер вызывается только в том случае, если аватар есть, так что здесь ничего ломаться не будет

Reply:
Ну как знаешь, но я всё-таки сделал бы проверку. Просто в ПР с тестами валится тест из-за того, что undefined попадает в место с '!'. Но это в файлах игры.

Reply:
этот конвертер вызывается после успешной валидации поля. я не знаю, как там тесты проверяют, но игру гляну

Reply:
Смотри сразу ПР с тестами, там поймёшь. Запусти у себя